tests/test_manifest.py

```diff
@@ -111,7 +111,8 @@ def test_sdist_includes_required_files(self):
 		]
 		for expected_file in expected_files:
 			self.assertIn(
-				expected_file, members,
+				expected_file,
+				members,
 				str("""Missing {expected} in sdist.""").format(expected=expected_file)
```

Comment:
This could also be improved as per #134 with F-strings.

```suggestion
			f"Missing {str(expected_file)} in sdist."
```